circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  cache_directories:
+    - "~/nilearn_data"
+  override:
+    - pip install --upgrade pip
+    - pip install -e .
+    - pip install sphinx matplotlib coverage Pillow
+
+test:
+  override:
+    - make
+    - cd doc && make
```

Comment:
you make want to use make html here. It seems that make all does html-noplot. Full disclosure I always use make html ...